src/Speckle.Objects/GIS/CRS.cs

```diff
@@ -5,9 +5,9 @@ namespace Speckle.Objects.GIS;
 [SpeckleType("Objects.GIS.CRS")]
 public class CRS : Base
 {
```

Comment:
wondering if we need a specific class for this, or if it would make more sense to attach these properties to the layer collection if that's the only place CRS is used.

Reply:
the plan is to move this to root eventually, and it needs required properties that we cannot enforce on Layer (or anywhere else)

Reply:
WKT (Well Known Text) gives all the information about the coordinate reference system (which is required). Name is also required, and Authority ID is useful, but not always available (e.g. for custom coordinate systems, there are no authority ID). UnitsNative are helpful if some of the properties on the features are stored in the original layer units (before reprojecting and sending to Speckle). 

Let me think about offsets and rotation. These are the arguments supported in QGIS with custom functionality we added, but not yet implemented in ArcGIS